RecoTracker/MkFitCore/src/MkFitter.h

```diff
@@ -0,0 +1,89 @@
+#ifndef RecoTracker_MkFitCore_src_MkFitter_h
+#define RecoTracker_MkFitCore_src_MkFitter_h
+
+#include "MkBase.h"
+
+#include "RecoTracker/MkFitCore/interface/HitStructures.h"
+
+namespace mkfit {
+
+  class CandCloner;
+
+  const int MPlexHitIdxMax = 16;
+  using MPlexHitIdx = Matriplex::Matriplex<int, MPlexHitIdxMax, 1, NN>;
+  using MPlexQHoT = Matriplex::Matriplex<HitOnTrack, 1, 1, NN>;
+
+  class MkFitter : public MkBase {
+  public:
+    MPlexQF Chi2;
+
+    MPlexHS msErr[Config::nMaxTrkHits];
+    MPlexHV msPar[Config::nMaxTrkHits];
+
+    MPlexQI Label;    //this is the seed index in global seed vector (for MC truth match)
```

Comment:
use CMS naming convention for data member

Reply:
Addressed in Round 4 follow-up PR trackreco/cmssw#79 commit: https://github.com/trackreco/cmssw/pull/79/commits/85c149fb2b8af53d752e1a334f4d90feed9ba2ef